app/helpers/complaints_helper.rb

```diff
@@ -1,7 +1,21 @@
 # frozen_string_literal: true
 
 module ComplaintsHelper
+  STATUS_CSS = { created: "is-dark",
```

Comment:
You don't have to worry about this right now, but constant name is singular while it represents a collection
I'd use a pluralize the name instead